source/_components/google_assistant.markdown

```diff
@@ -102,7 +102,7 @@ entity_config:
           required: false
           type: list
         type:
-          description: Override the domain how Google Assistant has to interpretet the entity. For example, set to `light` for a switch entity to have it be handeld as a light.
+          description: Override how Google Assistant will interpretet the domain of the entity. For example, set to `light` for a switch entity to have it be handled as a light.
```

Comment:
`Override how Google Assistant interprets the domain of the entity. For example, set to `light` for a switch entity to be handled as a light